### ISSUE TITLE
Issue when creating module

### ISSUE THREAD

**Issue by radko93** [patrickkabwe/create-nitro-module#17]
When creating a module with yarn I get
```✖ Failed to create nitro module: ENOENT: no such file or directory, open '/Users/xxx/.config/yarn/global/node_modules/create-nitro-module/assets/react-native.config.js'```

The assets folder is not in that folder
```
❯ ls /Users/xxx/.config/yarn/global/node_modules/create-nitro-module/
README.md	lib		node_modules	package.json
```

**Comment by patrickkabwe** [comment]
@radko93 thanks for opening this issue. i have resolved this issue in this PR #18

**Comment by radko93** [comment]
Thank you, I will test tomorrow.
